src/object_models.c

```diff
@@ -67,7 +67,7 @@ void func_8005F850(void) {
         checksum += *(u8 *)(((s32)&func_80024D54) + i);
     }
     if (checksum != gFunc80024D54Checksum) {
```

Comment:
Can you rename `gFunc80024D54Checksum` to something matching the new function `antipiracy_modify_surface_traction_table`?